### DISCUSSION TITLE
Date parsing

### DISCUSSION THREAD

**Discussion by vincent-tarrit** [filamentphp/filament#5960] [Help]
Hello, 

I want to show a date picker as 

```php
 DateTimePicker::make('start_at')
                ->required()
                ->label('Date de début')
                ->withoutSeconds(),
```

My date to parse is : 2023-04-01 08:00:00

But in the field it's 4th january 2023. However, Carbon parse it well as 1st april.

Do you have some idea for me ? Thank you !

**Comment by ArtMin96** [comment]
maybe
```php
->format('Y-m-d H:i')
```
